recipes-graphics/userland/userland_git.bb

```diff
@@ -67,6 +67,7 @@ EXTRA_OECMAKE_append_aarch64 = " -DARM64=ON "
 PACKAGECONFIG ?= "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'wayland', '', d)}"
 
 PACKAGECONFIG[wayland] = "-DBUILD_WAYLAND=TRUE -DWAYLAND_SCANNER_EXECUTABLE:FILEPATH=${STAGING_BINDIR_NATIVE}/wayland-scanner,,wayland-native wayland"
+PACKAGECONFIG[allapps] = ""
```

Comment:
I think it would be better to define it like 

`PACKAGECONFIG[allapps] = "-DALL_APPS=true,,,"`

and remove using
`ALLAPPS = "${@bb.utils.contains('PACKAGECONFIG', 'allapps', '-DALL_APPS=true', '', d)}"`

Reply:
D'oh! Of course that's how I should have done it in the first place!